### ISSUE TITLE
Backdrop 1.12.5

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#3601]
## Pre-release tasks

- [x] Merge commits @quicksketch
- [x] Create a new bugfix release @quicksketch
- [x] Create the next bugfix milestone @quicksketch
- [x] Move all unfinished issues to the next bugfix release milestone @klonos  
- [x] Draft Release notes @quicksketch
  - [x] Short, descriptive summary of the release
  - [x] Note if any changes were made to files outside the `core` directory
  - [x] Note if updates (update.php) needs to be run
  - [x] Include changelog since last version (generated with drush rn)
- [x] Draft Security Advisories @klonos
  * [x] https://backdropcms.org/security/backdrop-sa-core-2019-008

## Release tasks

- [x] Update bootstrap.inc with version number  @quicksketch
- [x] Tag for release, and push tag to GitHub  @quicksketch
- [x] Revert version number back  @quicksketch
- [x] Create release notes on GitHub @quicksketch
- [x] Update the front page download link on b.org @klonos
- [x] Tweet that a new release is out @stpaultim 

If this is a security release:
- [x] Publish Security Advisories on b.org @klonos
- [x] Mark the release node on b.org as a security release @klonos
- [ ] Request CVE @quicksketch

## Post-release tasks

- [x] Push the bug-fix release to the [Pantheon Backdrop repository](https://github.com/backdrop-ops/backdrop-pantheon)  @quicksketch
- [ ] Push the bug-fix release to the [Platform.sh Backdrop repository](https://github.com/platformsh/platformsh-example-backdrop) @jenlampton
- [ ] Push the bug-fix release to the [Docker Backdrop repository](https://github.com/docker-library/official-images/blob/master/library/backdrop) @jenlampton
- [ ] Push the bug-fix release to the [TugBoat Backdrop repository](https://github.com/backdrop-ops/backdrop-tugboat)  @quicksketch
- [ ] Update the Wikipedia articles @klonos
  - [ ] https://en.wikipedia.org/wiki/Backdrop_CMS
  - [ ] https://en.wikipedia.org/wiki/List_of_content_management_systems
- [ ] Update the Security Advisory with CVE  @quicksketch

**Comment by quicksketch** [comment]
Tag is up at https://github.com/backdrop/backdrop/releases/tag/1.12.5

**Comment by quicksketch** [comment]
For release notes, the other changes besides the security fix for 1.12.x were the following:

```
Issue #3575: Fix docblock in page.tpl.php
Issue #3554: [UX] Basis: Fix RTL breadcrumb style.
Issue #3543: [UX] Fix RTL password field style.
```
